features/admin.core.v1/configs/app.ts

```diff
@@ -82,8 +82,10 @@ export class Config {
             }
         }
 
-        if (skipAuthzRuntimePath) {
-            return window[ "AppUtils" ]?.getConfig()?.serverOriginWithTenant?.replace("/o/", "");
+        const serverOriginWithTenant: string = window[ "AppUtils" ]?.getConfig()?.serverOriginWithTenant;
+
+        if (skipAuthzRuntimePath && serverOriginWithTenant?.slice(-2) === "/o") {
```

Comment:
Don't we have this `/o` stored in a constant?

Reply:
Yes, we don't have that defined as a constant.

Anyways, this original condition is temporary hack which is to be removed with the introduction of new runtime. Hence resolving the review.